.github/CONTRIBUTING.md

```diff
@@ -333,50 +374,45 @@ Here are some contribution quick tips:
 - [`@carbon/themes`](https://github.com/carbon-design-system/carbon/tree/main/packages/themes)
 - [`@carbon/type`](https://github.com/carbon-design-system/carbon/tree/main/packages/type)
 
-And more! You can view a complete list of our packages
+And more! See a complete list
 [here](https://github.com/carbon-design-system/carbon/tree/main/packages).
 
-### Besides some of the obvious contributions mentioned above, how else can I contribute?
+### How else can I contribute?
+
+Great question! You can contribute without directly creating or maintaining
+assets:
 
-Great question! Contribution does not require creating or maintaining our
-assets. Here are some other ways you can contribute, which don't require much
-work:
+<!--
+TODO: Are these the internal Slack channels? If so, why are they listed here?
+-->
```

Comment:
We have to bridge the gap of both internal and external. It might be better to have like 

> For IBMers, join our internal slack community...

and then we could add discord for external

> Join our discord ...

The link is in the readme I think